src/Plugin/Field/FieldWidget/TextEDTFWidget.php

```diff
@@ -43,11 +44,11 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
       '#type' => 'checkbox',
       '#title' => $this->t('Ensure provided date values are valid.'),
       '#description' => $this->t(
-        'Negative dates, and the level 1 features unspecified dates, '.
-        'extended years, and seasons '.
-        'are not supported with strict date checking.<br />'.
-        'Uncertain/Approximate dates will have their markers removed before '.
-        'checking. (For example, "1984~?" will be checked as "1984".)'),
+        'Negative dates, and the level 1 features unspecified dates, 
+        extended years, and seasons
+        are not supported with strict date checking.<br />
```

Comment:
@dannylamb I missed this. Can't have the `<br />` in here. Probably this whole thing needs to be more like:
```
'#description' => $this->t(
  'Negative dates, and the level 1 features unspecified dates,
  extended years, and seasons are not supported with strict date 
  checking.') .
'<br />' .
t('Uncertain/Approximate dates will have their markers removed before
  checking. (For example, "1984~?" will be checked as "1984".)'),

Reply:
@whikloj I'll see if that passes.